dnf-behave-tests/dnf/cache.feature

```diff
@@ -11,16 +11,12 @@ Scenario: Do not error out when fail to load/store expired_repos cache
         """
     And I execute "chmod 777 /tmp/dnf/"
     And I execute "chmod 000 /tmp/dnf/expired_repos.json"
-   When I execute dnf with args "repoquery --setopt=cachedir=/tmp/dnf/" as an unprivileged user
+   When I execute dnf with args "repoquery --setopt=cachedir=/tmp/dnf/ dummy" as an unprivileged user
    Then the exit code is 0
     And stdout is
         """
         dummy-1:1.0-1.src
         dummy-1:1.0-1.x86_64
-        weird-1:1.0-1.src
-        weird-1:1.0-1.x86_64
-        weird-1:2.0-1.src
-        weird-1:2.0-1.x86_64
```

Comment:
The original intent with the miscellaneous repo was to not add "random packages" into it, but to have a very simple repo that will not change. Granted it's misnamed and the name suggests the purpose for which you've used it.

I'm not against adding the rpm to it, but we should have a repo to which we don't add packages and can be used for the majority of tests that just needs "something"...

Reply:
Oh, OK. I thought that miscellaneous was supposed for those kind of "special" packages (when you do not want to create a repo for them), and "simple-base" and "simple-updates" repos for those tests that just need any repo.

Reply:
Yeah that makes sense, I missed that there are simple-base and simple-updates now.

Wouldn't it make more sense to switch this scenario to use simple-base then? It doesn't have a special package in the miscellaneous repo.

Reply:
I switch the test to simple-base repo.